File: php/truesearch.php
<?PHP

$APIToken = ""; // PHPHive Truecaller API Token, Obtain it from https://tcapi.phphive.info/console/

$no = ""; // Any Number You want to Search 

search($no); // echo Details like Name, Internet Address ( including Profile Picutre, Email, Facebook & Twitter ), Spam Score, Spammer Type etc.

function search($no){

$uri = 'https://tcapi.phphive.info/'.$APIToken.'/search/'.$no;
$ch = curl_init($uri);
curl_setopt_array($ch, array(
    CURLOPT_HTTPHEADER  => array('X-User: PHPHive'),
    CURLOPT_RETURNTRANSFER  =>true,
 CURLOPT_SSL_VERIFYHOST => false,
 CURLOPT_SSL_VERIFYPEER => false,
    CURLOPT_VERBOSE     => 1
));
$response = curl_exec($ch);
curl_close($ch);
echo $response;

}

?>

File: python/truesearch.py
import urllib2

APIToken = "" # PHPHive Truecaller API Token, Obtain it from https://tcapi.phphive.info/console/
no = "" # Any Number You Want to Search

request_headers = {
"X-User": "PHPHive"
}

# For Searching User Details
print "Searching for "+no;
request = urllib2.Request("https://tcapi.phphive.info/"+APIToken+"/search/"+no, headers=request_headers)
contents = urllib2.urlopen(request).read()
print contents
